testing/chain.go

```diff
@@ -663,3 +625,27 @@ func (chain *TestChain) DeleteKey(key []byte) {
 	kvStore := chain.GetContext().KVStore(storeKey)
 	kvStore.Delete(key)
 }
+
+// IBCClientHeader will construct a 07-tendermint Header to update the light client
+// on the counterparty chain. The trustedHeight must be passed in as a non-zero height.
+func (chain *TestChain) IBCClientHeader(header *ibctm.Header, trustedHeight clienttypes.Height) (*ibctm.Header, error) {
```

Comment:
given that we don't currently store historical headers (we only keep the latest committed header), we could drop the header arg and assume that. But fine as is, might be more flexible for future testing scenarios